plugin/trino-thrift/src/test/java/io/trino/plugin/thrift/integration/TestThriftDistributedQueries.java

```diff
@@ -1,31 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.thrift.integration;
-
```

Comment:
@HiLany  Why did you delete this test?
Was this intentional?

Reply:
see #7117.

We need to merge TestThriftIntegrationSmokeTest and TestThriftDistributedQueries to form TestThriftConnectorTest. 
and `BaseConnectorTest` is also an implementation class of `AbstractTestQueries`.

Should we keep `TestThriftDistributedQueries`?

Reply:
@HiLany  I see. Thanks for pointing this out. You are right. The test is not anymore needed in the context of your PR